### PR TITLE
fix: physical table statistics info

### PR DESCRIPTION
**Pull request by killme2008**
I hereby agree to the terms of the [GreptimeDB CLA](https://github.com/GreptimeTeam/.github/blob/main/CLA.md).

## What's changed and what's your intention?
Fix table statistics info not working for the physical table in the metric engine.

## Checklist

- [ ] I have written the necessary rustdoc comments.
- [ ] I have added the necessary unit tests and integration tests.
- [ ] This PR requires documentation updates.

### CODE REVIEW

File: src/common/meta/src/datanode.rs [hunk 1]
```diff
@@ -77,7 +77,7 @@ pub struct RegionStat {
     pub rcus: i64,
     /// The write capacity units during this period
     pub wcus: i64,
-    /// Approximate bytes of this region
+    /// Approximate disk bytes of this region, including sst, index, manifest and wal
```
Comment (WenyXu) [inline src/common/meta/src/datanode.rs:80]: Maybe the data size will be a better word. We store data on S3 or disk.

Comment (killme2008) [reply src/common/meta/src/datanode.rs:80]: That's acceptable. Just a hint.

File: src/catalog/src/system_schema/information_schema/tables.rs [hunk 2]
```diff
@@ -259,7 +260,13 @@ impl InformationSchemaTablesBuilder {
                 let table_info = table.table_info();
 
                 // TODO(dennis): make it working for metric engine
-                let table_region_stats = if table_info.meta.engine == MITO_ENGINE {
+                let table_region_stats = if table_info.meta.engine == MITO_ENGINE
+                    || table_info
+                        .meta
+                        .options
+                        .extra_options
+                        .contains_key(PHYSICAL_TABLE_METADATA_KEY)
```
Comment (v0y4g3r) [inline src/catalog/src/system_schema/information_schema/tables.rs:268]: Can we add a method to `TableInfo` to check if a table is a "physical table"?

**Review by WenyXu — APPROVED** [review]

File: src/catalog/src/system_schema/information_schema/tables.rs (hunk 2 above)
Comment (killme2008) [reply src/catalog/src/system_schema/information_schema/tables.rs]: Refactor at https://github.com/GreptimeTeam/greptimedb/pull/4975/commits/9bcf284bccba200bd56946a1214b418809573e0b

**Review by v0y4g3r — APPROVED** [review]
LGTM

**Pull request merged**